### DISCUSSION TITLE
CLI examples don't work in windows

### DISCUSSION THREAD

**Discussion by marcosnils** [dagger/dagger#12491] [Help]
CLI examples: 

https://github.com/dagger/dagger/blob/main/cmd/dagger/query.go#L29-L41
https://github.com/dagger/dagger/blob/main/cmd/dagger/exec.go#L21-L24

Are unix specific, we should change it to depending on which platform the CLI is built, examples differ.

**Comment by mircubed** [comment]
@kpenfound is this still true? Is this something you can help with?
